cloudsync/tests/test_cs.py

```diff
@@ -57,21 +66,32 @@ def _fixture_scs(mock_provider_generator, mock_provider_creator, storage=None):
     cs.providers[LOCAL].name += '-l'
     cs.providers[REMOTE].name += '-r'
 
+    cs.nmgr.start()
+
     yield cs
 
+    cs.nmgr.stop()
     cs.done()
 
 
+@pytest.fixture(name="cs_nmgr")
+def fixture_cs_no_start(mock_provider_tuple):
+    yield from _fixture_cs(mock_provider_tuple, with_nmgr=True)
+
 @pytest.fixture(name="cs")
 def fixture_cs(mock_provider_tuple):
     yield from _fixture_cs(mock_provider_tuple)
 
 
-def _fixture_cs(providers, storage=None):
+def _fixture_cs(providers, with_nmgr=False, storage=None):
```

Comment:
curious, why do we need the with_nmgr option?

Reply:
Some tests explicitly start the cs, which also causes the notification manager to start during cs startup. If I start the notification manager in the fixture for those tests, then starting the cs in those tests will fail because double-starting the notification mgr will raise an exception. Those tests that want a notification manager, but don't want to start the cs, would use the cs_nmgr fixture, otherwise the regular cs fixture is fine